pvanalytics/quality/irradiance.py

```diff
@@ -233,13 +252,19 @@ def check_irradiance_limits_qcrad(solar_zenith, dni_extra, ghi=None, dhi=None,
 
     References
     ----------
-    .. [1] C. N. Long and Y. Shi, An Automated Quality Assessment and Control
-       Algorithm for Surface Radiation Measurements, The Open Atmospheric
-       Science Journal 2, pp. 23-37, 2008.
-
-    """
-    if not limits:
-        limits = QCRAD_LIMITS
+    .. [1] C. N. Long and Y. Shi, "An Automated Quality Assessment and Control
+       Algorithm for Surface Radiation Measurements," The Open Atmospheric
+       Science Journal, vol. 2, no. 1. Bentham Science Publishers Ltd.,
+       pp. 23–37, Apr. 18, 2008. :doi:`10.2174/1874282300802010023`.
+    .. [2]  C. N. Long and E. G. Dutton, "BSRN Global Network recommended QC
+       tests, V2.0," Baseline Surface Radiation Network (BSRN),
+       Accessed: Oct. 24, 2024. [Online.] Available:
+       <https://bsrn.awi.de/fileadmin/user_upload/bsrn.awi.de/Publications/BSRN_recommended_QC_tests_V2.pdf>_
```

Comment:
Not sure what's intended with this URL formatting, but the `<>_` characters aren't doing anything useful as it stands now